app/code/Magento/Cookie/Model/App/Response/HttpPlugin.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Cookie\Model\App\Response;
+
+use Magento\Framework\App\Response\Http;
+use Magento\Framework\Session\SessionManagerInterface;
+
+/**
+ * HTTP response plugin for frontend.
+ */
+class HttpPlugin
+{
+    /**
+     * @var SessionManagerInterface
+     */
+    private $sessionManager;
+
+    public function __construct(
+        SessionManagerInterface $sessionManager
+    ) {
+        $this->sessionManager = $sessionManager;
```

Comment:
Dependency which is not used anywhere in the class should not be injected.
It is a good idea to look for an alternative solution.